claim/models.py

```diff
@@ -1,17 +1,57 @@
 import datetime
 import json
 
+from django.db.models import Q
 from django.db import models
 from django.contrib.auth.models import User
-from django.conf import settings
 from django.db.models import Count
 from django.utils.translation import ugettext as _
 
 
+class Organization(models.Model):
+    id = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=255)
+
+    responsible_email = models.CharField(max_length=255)
```

Comment:
As i understand you ask to create some:

class OrganizationObserver(models.Model):
    observer_name =...
    observer_email =...
    ....

class Organization(modelsModel):
    observer = models.ForeighKey(OrganizationObserver)
    ....

?